.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Pull request
+about: Create a pull request for an issue in Altinn Studio
+---
+
+Thank you for contributing to Altinn:) We know this isn't the fun part, 
+but please make sure you follow our [contributing guidelines](../../CONTRIBUTING.md) and 
+put the same effort into the pull request as you did into the code and 
+it should soon find it's way to master.
+
+# {PR title}
+
+Summary of the changes (max 80 characters)
+
+## Description
+
+{Detail}
+
+## Fixes
+- #{issue number}
+
+## Verification
+- [ ] **Your** code builds clean without any errors or warnings
+- [ ] Manual testing done (required)
+- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
+- [ ] All tests run green
+- [ ] Documentation is updated
```

Comment:
- [ ] Documentation is updated with a separate linked PR in altinn-docs **(if applicable)**